templates/zerver/help/upload_share_files.md

```diff
@@ -0,0 +1,26 @@
+## Upload and Share Files
```

Comment:
Uploading and Sharing Files
- The help docs mostly uses present progressive for titles.